src/components/contact/ContactIconsList.tsx

```diff
@@ -0,0 +1,40 @@
+import { Text, Box, Stack } from '@mantine/core';
+import { IconSun, IconPhone, IconMapPin, IconAt } from '@tabler/icons-react';
+import './ContactIcons.scss';
+
+interface ContactIconProps
+  extends Omit<React.ComponentPropsWithoutRef<'div'>, 'title'> {
+  icon: typeof IconSun;
+  title: React.ReactNode;
+  description: React.ReactNode;
+}
+
+const ContactIcon = (props : ContactIconProps) => {
+  return (
+    <div className="wrapper">
+      <Box mr="md">
+        <props.icon size={24} />
+      </Box>
+      <div>
+        <Text size="xs" className="title">
+          {props.title}
+        </Text>
+        <Text className="description">{props.description}</Text>
+      </div>
+    </div>
+  );
+}
```

Comment:
This ContactIcon will never can be used at another location ? if yes move it in component